src/utilities/stripIgnoredCharacters.js

```diff
@@ -76,6 +76,9 @@ export function stripIgnoredCharacters(source: string | Source): string {
 
   let wasLastAddedTokenNonPunctuator = false;
   while (lexer.advance().kind !== TokenKind.EOF) {
+    if (lexer.token.kind === TokenKind.COMMENT) {
+      continue;
+    }
```

Comment:
nit: empty line will improve readability

Reply:
I tried to do it, but prettier won't allow me, all the files are automatically linted by VSCode... When i disabled auto format on save ```yarn test``` came up with a `prettier error`

Reply:
@nveenjain Strange prettier should always keep single empty line:
https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0AE8AzjABQCU+wAOlPvgO4AWOANnPqewB5wBOAOgCGAEwBuQrHAoCA1jigj8AQgC8q-ABUIshAGkFIgQFEA8gDFKNOvXw5UnHvwEwdCOYfzqN23VAOKAgDCpgCyocYAcppWtLa2kLAK6HAA3HG2AL4Z9Dn4AEasQoVC6TbZUBUgADQgEAAOuNBEyKBCfHwQDAAK7QgtKEJiEDgiNSD5fEJgujAAyvXTCgDmyDB8KbVMMAC2rADqLMSLYHBz-Ti4YpcAnsjgRC21CkT8MN1TyztCyKhCrK9agArIjcABCUxmcHmQh2cAAMgo4L9-oCQCDuHMVuwAIroCDwFEAuC1RZ8V58e75YpwVjjep8BQwfajGBMZAADgADKTOq99lN6vcGXAKWJkbUAI74+AfBoDEBCIgAWigcDgIg14z4cGlOB1HyEXx+SD+xNqrx2ODWGxJICI2LgeIJyNNqLtMGKLJEbOQACZaushGwVkEIDtvvcoNAJSB0K9NMUBma0WIUgBJRQIeZgRmNACCijmMBu7CJr0ymSAA

Reply:
Oh sorry I thought you meant:
```js
if (lexer.token.kind === TokenKind.COMMENT) {

      continue;
}
```